py/desitarget/mock/build.py

```diff
@@ -838,13 +838,11 @@ def _update_spectra_status(result):
                 os.rename(skyfile+'.tmp', skyfile)
 
         # Write out the dark- and bright-time standard stars.
-        for stdsuffix, stdbit in zip(('dark', 'bright'), ('STD_FSTAR', 'STD_BRIGHT')):
+        for stdsuffix, stdbit in zip(('dark', 'bright'), ('STD_FSTAR', 'STD_BRIGHT', 'STD_WD')):
```

Comment:
This zip has two elements in the first list and 3 in the second list, and thus will drop the 3rd element as currently written.  This logic may need a slightly larger refactor to have more than one target bit mapping to the "dark" or "bright" standards.